Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -9,6 +9,7 @@
 import UIKit
 import SafariServices
 import Crashlytics
+import ViewAnimator
 
 class NewsTableViewController: UITableViewController,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/7343f4625b53ffff94cd9e5a6b4dab4ffcc3bea5/Headlines/src/Controllers/NewsTableViewController.swift#L14"></span>Type body should span 200 lines or less excluding comments and whitespace: currently spans 243 lines</td>
    </tr>
  </tbody>
</table>